tampere/meetup.yaml

```diff
@@ -129,7 +129,7 @@ meetups:
     - luxaslabs
     - cncf
     venue: cybercom
-members: 150
```

Comment:
please drop this change, I fixed the cause of this being an issue at latest master